### DISCUSSION TITLE
creating 2 lists, causes the page to not work properly

### DISCUSSION THREAD

**Discussion by obovoid** [community/community#79376] [Other Feature Feedback, Questions, & Ideas] [answered]
The bug mentioned happens in the repositories tab where you can not only star your projects but also add them to a list (with the arrow pointing down next to the star)

The Issue:
Creating Lists only works the first time.
You create a list by typing a title, optionally a description and then click on the create button.

Expected behavior:
List gets created, The "modal page" closes. Project gets added to the list.

Behavior now:
If you create it the first time ever (for one project) it works fine
As soon as you create a second list on the same project and you click on create then you have an endless load from the button saying "creating..."
Reloading the page, closes the modal and adds the project to the created list.
But reloading does not fix the issue, endless loads are still happening everytime you create a list and can only be fixed by reloading the page. Clearing the cache of the page also has no effect (ctrl + F5)
Choosing a different project and creating a list seems to work as long as there is no other list active already.

And as a side node:
I don't know if and how to delete created lists.
I have 7 lists, all called test 1 ..7 that I do not want or need.

Thanks for reading,
Cheers.

Tested on Chromium Version 120.0.6099.110

**Comment by oasissoman** [comment]
> The bug mentioned happens in the repositories tab where you can not only star your projects but also add them to a list (with the arrow pointing down next to the star)
> 
> The Issue:
> Creating Lists only works the first time.
> You create a list by typing a title, optionally a description and then click on the create button.
> 
> Expected behavior:
> List gets created, The "modal page" closes. Project gets added to the list.
> 
> Behavior now:
> If you create it the first time ever (for one project) it works fine
> As soon as you create a second list on the same project and you click on create then you have an endless load from the button saying "creating..."
> Reloading the page, closes the modal and adds the project to the created list.
> But reloading does not fix the issue, endless loads are still happening everytime you create a list and can only be fixed by reloading the page. Clearing the cache of the page also has no effect (ctrl + F5)
> Choosing a different project and creating a list seems to work as long as there is no other list active already.
> 
> And as a side node:
> I don't know if and how to delete created lists.
> I have 7 lists, all called test 1 ..7 that I do not want or need.
> 
> Thanks for reading,
> Cheers.
> 
> Tested on Chromium Version 120.0.6099.110

- @mgriffin ![transcript.txt](https://github.com/community/community/files/13850884/transcript.txt)
![2021 IWFAQRP Annual Report.pdf](https://github.com/community/community/files/13851072/2021.IWFAQRP.Annual.Report.pdf)

- []()_@oasissoman _

**Comment by obovoid** [comment, marked as answer]
Lists no longer have an endless load. Deleting lists also works with the edit button.
